src/test/erasure-code/ceph_erasure_code_benchmark.cc

```diff
@@ -123,14 +123,23 @@ int ErasureCodeBench::setup(int argc, char** argv) {
 
   k = atoi(profile["k"].c_str());
   m = atoi(profile["m"].c_str());
+
+  if ("lrc" == plugin)
+    l = atoi(profile["l"].c_str());
+  else
+    l = 0;
   
   if (k <= 0) {
     cout << "parameter k is " << k << ". But k needs to be > 0." << endl;
     return -EINVAL;
   } else if ( m < 0 ) {
     cout << "parameter m is " << m << ". But m needs to be >= 0." << endl;
     return -EINVAL;
-  } 
+  } else if ((l != 0) && (k+m) % l) {
+    cout << "parameter l is " << l << ". But k+m needs to be multiple of l."
```

Comment:
this is not necessary: the `ErasureCodePluginLrc::init()` will take care of this. see `ErasureCodeLrc::parse_kml()`.